lib/python/treadmill_aws/hostmanager.py

```diff
@@ -5,6 +5,8 @@
 import time
 import yaml
 
+from botocore.exceptions import ClientError
```

Comment:
we do not import classes, - from botocore import exceptions as botoexc something like that.

Reply:
Fixed in subsequent PR.